homework_7/functional.py

```diff
@@ -0,0 +1,72 @@
+import sys
+
+def sequential_map(*args):
+    """Takes several functions and a container and returns a list 
+    of all container values after sequentional applying of functions to them"""
+    funcs = args[:-1]
+    container = args[-1]
+    for func in funcs:
+        container = list(map(func, container))
+    return container
```

Comment:
Превратить элемент в список можно только в самом конце, результат map можно передавать в другие map'ы
```suggestion
        container = map(func, container)
    return list(container)
```